paasta_tools/frameworks/native_service_config.py

```diff
@@ -171,7 +171,10 @@ def base_task(
         computed separately.
         """
         docker_volumes = self.get_volumes(
-            system_volumes=system_paasta_config.get_volumes()
+            system_volumes=system_paasta_config.get_volumes(),
+            uses_bulkdata_default=system_paasta_config.config_dict.get(
+                "uses_bulkdata_default", True
+            ),
```

Comment:
this gets repeated a lot, and should probably be a method on SystemPaastaConfig.

I feel less strongly about this, but I think I'd also slightly prefer to see `get_volumes` just take the whole SystemPaastaConfig object as a parameter rather than all 3 callers having to pull two pieces of information from SPC.

Reply:
> I feel less strongly about this, but I think I'd also slightly prefer to see get_volumes just take the whole SystemPaastaConfig object as a parameter rather than all 3 callers having to pull two pieces of information from SPC.

Interesting, I did go down this route, and found a lot of tests failed as they were not setting up the `SystemPaastaConfig` object, and by [law of demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) was thinking perhaps the `InstanceConfig` object / `get_volumes` shouldn't need to know details about `SystemPaastaConfig`

Agree on making this a method on `SystemPaastaConfig` though

Reply:
@EvanKrall updated with a method on SystemPaastaConfig

Reply:
oh also worth noting - once we've rolled these changes out, theres no need for the `uses_bulkdata_default` param - we can hardcode it to `False`, so I can loop back and remove it